Adjust references to magic(4) to be magic(5) in the mkisofs/mkisofs.8 man page.

The patch was sent upstream. The change will be included in the next release.

--- a/mkisofs/mkisofs.8
+++ b/mkisofs/mkisofs.8
@@ -2899,7 +2899,7 @@ .PP
 The format of the
 .I magic
 file is almost identical to the
-.BR magic (4)
+.BR magic (5)
 file used by the Linux
 .BR file (1)
 command - the routines for reading and decoding the
@@ -2938,7 +2938,7 @@ 4/string/mdat/txtt MooV  QuickTime movie
 .TE
 .PP
 The format of the file is described in the
-.BR magic (4)
+.BR magic (5)
 man page. The only difference here is that for each entry in the magic file, the
 .I message
 for the initial offset
